Cargo.toml

```diff
@@ -18,7 +18,7 @@ byteorder = "1.0"
 crc32fast = "1.2.0"
 once_cell = "1.0"
 regex ={version = "1.3.0", default-features = false, features = ["std"]}
-tantivy-fst = "0.2"
+# tantivy-fst = "0.2"
```

Comment:
I merged and published tantivy-fst = 0.2.1.

Can you update this and remove `[dependencies.tantivy-fst]`